packages/paste-core/components/file-picker/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Default, Disabled, Customized} from '../stories/index.stories';
+
+describe('FilePicker', () => {
+  it('should render', () => {
+    const {getByText} = render(<Default data-testid="test-file-picker" />);
+    expect(getByText('Choose a file')).toBeDefined();
+  });
+  it('should render the file input in the DOM', () => {
```

Comment:
Suggestion(non-blocking): this test could be combined with the should render test. You'd just check if the text is there and that the input is there.